src/lib/scripts/milestoneLinks.ts

```diff
@@ -89,18 +95,50 @@ function getTelephoneLinkHtml(link: string, text: string): HTMLElement {
 function getAudioLinkHtml(
     link: string,
     text: string,
-    clipNumber: number
+    clipNumber: number,
+    onClickFunction: (e: any) => void
 ): [HTMLElement, HTMLElement] {
     const audio = document.createElement('audio');
     const audioId = 'audio' + padWithInitialZeros(clipNumber.toString(), 3);
-    const filename = 'clips/' + filenameWithoutPath(link);
+
+    const filename = filenameWithoutPath(link);
+    let src = '';
```

Comment:
@davidmoore1, Is it possible that the filename will not be in the list of audio files?  If so, this code is different then.